handlers/mapping_handlers.go

```diff
@@ -62,6 +66,28 @@ func (env *Env) PostMappingHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Check with Karen if user to be added exists
+	url := fmt.Sprintf("http://" + env.KarenHost + usersRoute + strconv.FormatInt(reqMember.UserID, 10))
+	request, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		internalServerError(w, err)
+		return
+	}
+	request.Header.Add("User_ID", strconv.FormatInt(reqMember.UserID, 10))
+	response, err := env.Client.Do(request)
+	if err != nil {
+		errMsg := "Karen GET request failed"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
```

Comment:
I think this can just be an `internalServerError()` since an error here would be unexpected.